pkg/apis/core/validation/validation.go

```diff
@@ -2429,9 +2437,9 @@ var validEnvDownwardAPIFieldPathExpressions = sets.NewString(
 	"status.podIPs",
 )
 
-var validContainerResourceFieldPathExpressions = sets.NewString("limits.cpu", "limits.memory", "limits.ephemeral-storage", "requests.cpu", "requests.memory", "requests.ephemeral-storage")
+var validContainerResourceFieldPathExpressions = sets.New("limits.cpu", "limits.memory", "limits.ephemeral-storage", "requests.cpu", "requests.memory", "requests.ephemeral-storage")
```

Comment:
Since you are touching these, can you linewrap like `supportedHostPathTypes` does?  Any place that has more than 2 values